src/Model/Filter/Value.php

```diff
@@ -1,6 +1,9 @@
 <?php
 namespace Search\Model\Filter;
 
+use Cake\Database\Expression\QueryExpression;
+use Search\Manager;
+
 class Value extends Base
```

Comment:
I would stay at `Value` otherwise we have an inconsistency between the usage of `Like` and `Value`. Where is the breaking change?